NEWS.md

```diff
@@ -331,7 +339,7 @@ of general interest to the broader R community. More will be included in future
 * `object_usage_linter` has been changed to ensure lint-position is indicated
   relative to the start of the file, rather than the start of a defining
   function (#432, @russHyde).
-* `commas_linter` now allows spaces to come before a comma when used to denote a 
+* `commas_linter` now allows spaces to come before a comma when used to denote a
```

Comment:
Do we want to touch old release notes?
I thought about e.g. increasing consistency by adding `()` to all functions, but not changing the wording in any material way.

Reply:
I think touch-up stuff like whitespace/typos are OK. Personally I don't think `()` is worth the effort, but happy to approve a PR.

Reply:
Allright. I'll keep this in mind whenever I do another pass.